src/containers/App.js

```diff
@@ -1,28 +1,27 @@
-import React, { Component} from 'react';
-import classes from './App.css';
+import React, { Component } from 'react'
+import classes from './App.css'
+import SearchApp from '../components/SearchApp/SearchApp'
 
 class App extends Component {
-  constructor(props) {
-    super(props);
-  }
-
   state = {
     persons: [
-      { name: 'Manu', age: 29 },
-      { name: 'Max', age: 25 },
-      { name: 'Stephanie', age: 30 },
+      { name: 'Manu' },
+      { name: 'Max' },
+      { name: 'Stephanie' },
+      { name: 'Sam' },
+      { name: 'alex stephanie' },
+      { name: 'Bob' },
+      { name: 'Bedrick' },
     ],
-    showPersons: false,
-  };
+  }
 
   render() {
-
     return (
       <div className={classes.App}>
-
+        <SearchApp persons={this.state.persons} />
```

Comment:
i don't understand why we have to create two app components. IMO search-app and app are the same thing